app/src/main/java/com/kwartracker/android/home/ui/DateRangeDialog.kt

```diff
@@ -0,0 +1,73 @@
+package com.kwartracker.android.home.ui
+
+import android.app.Dialog
+import android.content.DialogInterface
+import android.graphics.Color
+import android.graphics.drawable.ColorDrawable
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.view.WindowManager
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.databinding.DataBindingUtil
+import androidx.fragment.app.DialogFragment
+import com.google.android.material.bottomsheet.BottomSheetBehavior
+import com.kwartracker.android.R
+import com.kwartracker.android.databinding.DialogDateRangeHomeBinding
+
+class DateRangeDialog : DialogFragment() {
```

Comment:
@danacealfonso 
This should be a [BottomSheetDialogFragment](https://developer.android.com/reference/com/google/android/material/bottomsheet/BottomSheetDialogFragment).